Makefile

```
@@ -34,7 +34,15 @@ all: test
 
 publish: clean ci compile build push
 
-ci: check-headers gofmt govet golint test
+ci: check-headers check-unused gofmt govet golint test
+
+.PHONY: check-unused
+check-unused:
+	@echo "==> Running check for unused/missing packages in go.mod"
```

uber-nit pick: I think Go modules distinguishes between packages and modules:

```suggestion
	@echo "==> Running check for unused/missing modules in go.mod"
```